builder/dockerfile/builder.go

```diff
@@ -75,16 +75,21 @@ func (bm *BuildManager) Build(ctx context.Context, config backend.BuildConfig) (
 		ProgressWriter: config.ProgressWriter,
 		Backend:        bm.backend,
 		PathCache:      bm.pathCache,
+		ContainerBackend: NewContainerBackend(
+			config.ProgressWriter.StdoutFormatter,
```

Comment:
I think these should be passed with `Run`, there is no constraint that an exec output needs to end up in a central builder output.